core/setup.py

```diff
@@ -56,6 +56,7 @@
     'protobuf >= 3.0.0',
     'google-auth >= 0.4.0, < 2.0.0dev',
     'google-auth-httplib2',
+    'google-gax>=0.15.7, <0.16dev',
```

Comment:
1.  Do we want to do this? (probably yes)
1.  If we do this

    1. You should remove the dep from any subpackages that depend on `google-gax` (if they exist)
    1. Why do we `try/except` the imports?